src/Core/src/Handlers/Editor/EditorHandler.Windows.cs

```diff
@@ -22,6 +26,14 @@ protected override void DisconnectHandler(MauiTextBox nativeView)
 			nativeView.LostFocus -= OnLostFocus;
 		}
 
+		protected override void SetupDefaults(MauiTextBox nativeView)
+		{
+			_placeholderDefaultBrush = nativeView.PlaceholderForeground;
+			_defaultPlaceholderColorFocusBrush = nativeView.PlaceholderForegroundFocusBrush;
```

Comment:
One thing that needs to be checked, should we be setting the default, or unsetting/clearing?

Reply:
I don't think we should clear, because imagine that the user set a color via a winui xaml style , we want to set that inicial color not just clear the color